linker/Linker/Driver.cs

```diff
@@ -72,7 +74,65 @@ public static int Execute (string[] args, ILogger customLogger = null)
 
 		public Driver (string [] args)
 		{
-			_queue = new Queue<string> (args);
+			_queue = ProcessResponseFile (args);
+		}
+
+		Queue<String> ProcessResponseFile (string [] args)
```

Comment:
Can you make this public static and then add some unit tests?

Reply:
Fixed

Reply:
Why is it not static?